telemetry/src/main/java/com/newrelic/telemetry/TelemetryClient.java

```diff
@@ -68,6 +76,15 @@ public void sendBatch(SpanBatch batch) {
     scheduleBatchSend((b) -> spanBatchSender.sendBatch((SpanBatch) b), batch, 0, TimeUnit.SECONDS);
   }
 
+  /**
+   * Send a batch of metrics, with standard retry logic. This happens on a background thread,
```

Comment:
```suggestion
   * Send a batch of events, with standard retry logic. This happens on a background thread,
```

Reply:
This is resolved in the other PR :)

Reply:
aha. I got confused with the PR inception.